core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1075,12 +1075,23 @@ account.altcoin.popup.ZEC.msg=When using Zcash you can only use the transparent
 the z-addresses (private), because the arbitrator would not be able to verify the transaction with z-addresses.
 account.altcoin.popup.XZC.msg=When using Zcoin you can only use the transparent (traceable) addresses not \
 the untraceable addresses, because the arbitrator would not be able to verify the transaction with untraceable addresses at a block explorer.
-account.altcoin.popup.grin.msg=Bisq supports only the Grinbox (Wallet713) wallet URL format. Please be sure that you use the \
-  latest Grinbox release and understand the process of transferring and receiving GRIN. \
-  The GRIN sender is required to provide a proof that they sent GRIN successfully. If they cannot provide the proof a \
-  potential dispute will be resolved in favor to the GRIN receiver. \n\n\
+account.altcoin.popup.grin.msg=GRIN requires an interactive process between the sender and receiver to create the \
+  transaction. Be sure to follow the instructions from the GRIN project web page to reliably send and receive GRIN \
+  (the receiver need to be online or at least be online in a certain time frame). \n\n\
```

Comment:
(the receiver **needs** to be online or at least be online **during** a certain time frame)